src/Types.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 
 import * as React from 'react';
+import { Style } from '@modusoperandi/licit-custom-styles';
```

Comment:
If you're importing from custom styles, why not just import the runtime? Also, I don't see this in the peer dependency list.

Reply:
Have added Style type to avoid custom styles dependency